setup.py

```diff
@@ -22,6 +22,7 @@ def package_data(pkg, root_list):
     name='olx-utils',
     version='0.1.1',
     description='Utilities for edX OLX courses',
+    long_description=open('README.rst').read(),
```

Comment:
RST kinda sucks compared to markdown, but this is a very nice way to avoid redundancy, indeed.

Reply:
You can always write whatever you like in Markdown and then use pandoc (i.e. legit and true magic) to convert to RST.